plume-models/src/db_conn.rs

```diff
@@ -38,3 +38,15 @@ impl Deref for DbConn {
         &self.0
     }
 }
+
+// Execute a pragma for every new sqlite connection
+#[derive(Debug)]
+pub struct PragmaForeignKey;
+impl CustomizeConnection<Connection, ConnError> for PragmaForeignKey {
```

Comment:
oh, I was searching for exactly that but did not find it

Reply:
Looks like it is not working at all :upside_down_face:

Reply:
I think your changes are only affecting the web interface, which is at the same time the most important part, and the one not tested by travis :upside_down_face: 
I've tested doing similar changes in plume_model::lib:tests::db (adding `sql_query("PRAGMA foreign_keys = on;").execute(&conn).unwrap();`) which seems to work fine